.github/workflows/ci.yml

```diff
@@ -23,6 +23,6 @@ jobs:
       with:
         name: test-results
         path: |
-          build/reports/tests/**/*.*
-          */build/reports/tests/**/*.*
+          'build/reports/tests/**/*.*'
+          '*/build/reports/tests/**/*.*'
```

Comment:
@simonbasle These quotes should be removed...

https://github.com/reactor/reactor-kafka/runs/4499128439?check_suite_focus=true
<img width="882" alt="Screen Shot 2021-12-13 at 9 29 34 AM" src="https://user-images.githubusercontent.com/483832/145830671-30ec8eed-7496-452b-8032-1a55dbd36f09.png">

After I removed them, the test results were successfully captured...

https://github.com/reactor/reactor-kafka/actions/runs/1570293408#artifacts